configuration/coding_plans.py

```diff
@@ -68,6 +68,8 @@ def get_rqa_coding_plans(pipeline_name):
                    run_id_field="rqa_s01e03_run_id",
                    coda_filename="UNICEF_COVID19_SOM_s01e03.json",
                    icr_filename="rqa_s01e03.csv",
+                   katikati_survey_start_time="2020-07-24T14:00:00+03:00",
```

Comment:
I think this is fine to get us to the end of unicef/to urgently fix the problem of duplicated messages in Coda. However, it only allows us to set one survey per episode, which might not work on a project that has multiple surveys per episode. I wonder if it would be better to have a list of (start, end) pairs in the configuration json for when the surveys were taking place instead, or to work with KK to manage this in Rapid Pro by dropping people into a flow that captures their KK survey response after sending a message.

Reply:
The latter is better. I suggest it in the channel and didn't get any feedback. Updated to use list of (start, end) pairs in coding_plans.py so that we have the flexibility to filter within episodes.